docs/tutorials/anomaly-detection.rst

```diff
@@ -0,0 +1,542 @@
+Streaming anomaly detection with Apache Flink®, Apache Kafka® and PostgreSQL®
+==============================================================================================
+
+.. Note::
+
+    This tutorial doesn't assume any existing Apache Kafka®, PostgreSQL® or Apache Flink® knowledge
+
+What you will learn
+---------------------------
+
+The tutorial provides an example of a streaming anomaly detection system, by following it you'll learn:
+
+* How to create a fake streaming dataset
+* How to create and use Apache Kafka® for data streaming
+* How to create and use PostgreSQL® to store threshold data
+* How to create and use Apache Flink® to define streaming data pipelines
+* How to push the outcome of the anomaly detection system as a Slack notification
+
+
+What are you going to build
+---------------------------
+
+In this tutorial you will build a streaming anomaly detection system based on IoT sensor readings recording the CPU usage percentage for various devices. After setting up the various components and enabling the data flow, you'll then create a basic filtering pipeline to separate the usage values surpassing a fixed threshold (80%). 
+
+.. mermaid::
+
+    graph LR;
+
+        id1(IoT device)-- sensor reading -->id2(usage > 80%?);
```

Comment:
Can we have a small but meaningful example behind what sensor reading we're looking for?
It will be easier to follow, if it is something relatable. For example, you can suggest that the sensors measure the temperature or humidity across the building, or something else. And then follow that story.